drivers/adodb-sybase.inc.php

```diff
@@ -47,6 +47,12 @@ class ADODB_sybase extends ADOConnection {
 
 	var $port;
 
+	/** @var mixed */
```

Comment:
I only see this used as a bool (vs mixed), though its only read from and never set so I'm not sure